programs/ya.make

```diff
@@ -1,3 +1,27 @@
-RECURSE(
-    server
+PROGRAM(clickhouse)
+
+CFLAGS(
+    -DENABLE_CLICKHOUSE_CLIENT
+    -DENABLE_CLICKHOUSE_EXTRACT_FROM_CONFIG
+    -DENABLE_CLICKHOUSE_SERVER
```

Comment:
Why don't enable all?

Reply:
Only those are required to enable tests in Arcadia for now. I can add other instances later.